downstream/modules/viewing-cost-reports.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+// assembly-adding-ocp-sources.adoc
+:_content-type: PROCEDURE
+
+[id="viewing-cost-reports_{context}"]
+= Viewing your {product-title} reports
+
+[role="_abstract"]
+The +{operator}+ deletes your {product-title} reports upon successful upload to {platform}. These reports are then processed and deleted from the Red Hat servers. Therefore, {product-title} reports cannot be viewed when operating normally. However, you can install a duplicate instance of +{operator}+ in a restricted network configuration to view your reports. Alternatively, you can install the restricted network configuration to review normal report content before installing in a normal configuration. 
```

Comment:
I'm not too sure about the `Alternatively` statement. The idea there is to install the operator, set upload to `false`, let the reports generate for some time, copy the reports out of the operator, reset upload to `true` so that the operator ships the reports to console.redhat.com, and then reset upload back to `false` to repeat the process. It is certainly doable to view reports this way, but is likely error prone. Maybe if we are providing guidance on this, it is perhaps best to recommend 2 installations instead of fiddling with the `upload` setting.

Reply:
How long after you install the operator before it sends its first report? I was thinking about a use case where an org would need to submit the reports to their compliance department for a security review before using cost management.